### PR TITLE
add cfcfrj-legit-info to up-for-grabs

### PR DESCRIPTION
**Pull request by demilolu**

### CODE REVIEW

File: _data/projects/cfcfrj-legit-info.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Call for Code for Racial Justice: Legit-Info
```
Comment (ritwik12) [inline _data/projects/cfcfrj-legit-info.yml:1]: @demilolu  Please remove ":" from the line, being YML it causes issues.

File: _data/projects/cfcfrj-truth-loop.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Call for Code for Racial Justice: Truth Loop
```
Comment (ritwik12) [inline _data/projects/cfcfrj-truth-loop.yml:1]: @demilolu  Please remove ":" from the line, being YML it causes issues.

File: _data/projects/cfcfrj-legit-info.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+site: https://developer.ibm.com/callforcode/racial-justice/
+tags:
+  - bootstrap
+  - python
+  - django
+  - civic-hacking
+  - tech-for-good
+upforgrabs:
+  name: Call-for-Code-for-Racial-Justice-Legit-Info
+  link: https://github.com/Call-for-Code-for-Racial-Justice/Legit-Info
```
Comment (ritwik12) [inline _data/projects/cfcfrj-legit-info.yml:12]: Need label link here

File: _data/projects/cfcfrj-truth-loop.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+desc: Web app that allows users to find local legislation and upload their feedback on how the legislation affects them
+site: https://developer.ibm.com/callforcode/racial-justice/
+tags:
+  - react
+  - nodejs  
+  - civic-hacking
+  - tech-for-good
+upforgrabs:
+  name: Call-for-Code-for-Racial-Justice-Truth-Loop
+  link: https://github.com/Call-for-Code-for-Racial-Justice/Truth-Loop
```
Comment (ritwik12) [inline _data/projects/cfcfrj-truth-loop.yml:11]: Label link please

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by shiftkey** [comment]
Closing this out due to inactivity. 

@demilolu if you're still interested in adding the project please submit a new pull request and consider the feedback from this discussion.

**Pull request closed without merging**